css/main.css

```diff
@@ -6,6 +6,17 @@
 
 
 
+/**** Global styles for Acessibilty ****/
++
++body {
++   color: black;
++	font family: Arial, "Helvetica Neue", Helvetica, sans-serif;
++	font-size: 1em;
++	}
++
++a {
++   color: blue;
```

Comment:
While blue is a solid color that stands out, we have to keep in mind for visually impaired people, the text color has a high contrast with the background color.